### DISCUSSION TITLE
SageMaker, Azure Machine Learning, Streamlit Apps

### DISCUSSION THREAD

**Discussion by MexicanoAfortunado** [SuperDataScience-Community-Missions/MM0001-Operation-Mind-Shield#4] [Q&A]
Does anyone have experience with SageMaker, Azure ML, or Streamlit Apps?
I haven't used any of these tools and could use a walkthrough.

**Comment by ohinson01** [comment]
I have experience with Streamlit Apps, but not the other two. I haven't used many tools in AWS except for AWS EC2.

**Reply by MexicanoAfortunado** [reply]
Ok, cool.  When and if I get to that point, I'll ping you!

**Reply by ohinson01** [reply]
Sorry for the late response, but sounds good!
